client/src/models.ts

```diff
@@ -54,7 +54,7 @@ export interface ConsentConflictI {
   [propName: string]: any
 }
 export interface ConversionTypesI {
-  blurb?: string
+  blurb?: string[]
```

Comment:
Should rename to `blurbs`.

Also below.

Reply:
will do 👍

Reply:
Updated in next commit. A LOT of `blurb` converted to `blurbs`